bin/haraka

```diff
@@ -208,20 +208,19 @@ function copyFile (srcFile, dstFile) {
 
 function setupHostname (confPath) {
     mkDir(confPath);
-    const hlPath = path.join(confPath, 'host_list');
-    try {
-        if (fs.statSync(hlPath).isFile()) return;
-    }
-    catch (ignore) {}
 
-    const hostname = os.hostname() + "\n";
-    try {
-        const fd = fs.openSync(hlPath, 'w');
-        fs.writeSync(fd, hostname, null);
-    }
-    catch (e) {
-        warning("Unable to write config/host_list file: " + e);
-    }
+    const hostname = `${os.hostname()}${os.EOL}`;
+
+    ['me','host_list'].forEach(f => {
+        const cfPath = path.join(confPath, f);
+        try {
+            if (fs.statSync(cfPath).isFile()) return;
```

Comment:
Won't this fail if cfPath doesn't exist (because fs.statSync returns null, and then you try and do .isFile on it?) I think that was the point of the original version.

Reply:
Ahh, right you are. Fixed.